.github/workflows/onPushToMain.yml

```diff
@@ -0,0 +1,48 @@
+name: Push to main
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  tests:
+      uses: salesforcecli/github-workflows/.github/workflows/unitTest.yml@main
+  release:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    needs:
+    - tests
+    steps:
+    - uses: actions/download-artifact@v2
```

Comment:
This line we can remove, since we haven't used an upload-artifact action prior to this.